namer/namer.cc

```diff
@@ -1555,20 +1553,21 @@ class TreeSymbolizer {
             klass.symbol = ctx.owner.data(ctx)->enclosingClass(ctx).data(ctx)->lookupSingletonClass(ctx);
             ENFORCE(klass.symbol.exists());
         } else {
-            core::SymbolRef symbol = klass.symbol;
+            auto symbol = klass.symbol;
             if (symbol == core::Symbols::todo()) {
-                symbol = squashNames(ctx, ctx.owner.data(ctx)->enclosingClass(ctx), klass.name);
+                auto squashedSymbol = squashNames(ctx, ctx.owner.data(ctx)->enclosingClass(ctx), klass.name);
```

Comment:
OK so, `symbol` is a MethodRef, it can only become a class or module within this arm of the conditional so i moved it in.